repo-agent/Makefile

```diff
@@ -132,10 +132,14 @@ bin/key.json:
 	echo "Follow instructions in ./syncer/README.md"
 
 .PHONY: create-secrets
-create-secrets: bin/tls.crt bin/key.json
```

Comment:
we can still retain this target and print the warning there.

Reply:
done